proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -58,6 +58,12 @@ message FileInfo {
   string path = 1;
   bool is_dir = 2;
   repeated FileInfo children = 3;
+  FileOptions file_options = 4;
```

Comment:
Adding file options to this `message FileInfo` is not going to help us, unfortunately. If you take a look at `rpc GetPackageDocumentation`, you'll see that it doesn't return `FileInfo` anywhere, it only returns the messages, enumerations and services.

Each of the types has a couple of fields providing information about the file they are declared in:

```proto
  // file_path is the normalized path of the file containing the service.
  // This is used for navigating to the source code for the service.
  string file_path = 5;
  Location location = 6;
```

This is where file level deprecation is best added. 

Since package documentation is not concerned with files at all (it only needs the file path and location so that clients can link to source files), I do not think adding a `message FileOptions` is helpful. IMO, a plain `bool file_deprecated` field is sufficient.

Reply:
Thanks Timo. I missed that the FileInfo is not returned from the `rpc GetPackageDocumentation`. I agree that the `message FileOptions` is not necessary in that case, since the `file_deprecated` is added separately to each of the types.
I think `message FileOptions` will be necessary when we decide to render other options at the file level, but it's not part of the current scope, so I'll remove it.